.gitignore

```diff
@@ -1,6 +1,7 @@
 # Gradle build files
 /.gradle/
 /build/
+/bin/
```

Comment:
bin folder is still inside. Please fix it.

Reply:
Yup removed!